defold-rive/pluginsrc/com/defold/bob/pipeline/Rive.java

```diff
@@ -269,5 +450,49 @@ public static void main(String[] args) throws IOException {
 
 
         RIVE_UpdateVertices(p, 0.0f);
+
+        int count = 0;
+        RiveVertex[] vertices = RIVE_GetVertexBuffer(p);
+
+        System.out.printf("Vertices: count: %d  size: %d bytes\n", vertices.length, vertices.length>0 ? vertices.length * vertices[0].size() : 0);
```

Comment:
Maybe wrap these debug logs in a function or something, might get spammy? I guess we can do that later though

Reply:
Yeah, I thought I had removed all the logs...